src/services/notifications/email/systemMail/verifyEmail.tsx

```diff
@@ -15,7 +15,7 @@ export async function sendVerifyEmail(user: UserFiltered, email: string) {
         sub: user.id,
     }, emailValidationExpiration)
 
-    const link = `${process.env.DOMAIN}/register?token=${jwt}`
+    const link = `${process.env.NEXTAUTH_URL}/register?token=${jwt}`
```

Comment:
Really minor thing, but I'd consider depending on next auth specific env variables bad practice. If next auth changes the name of this env variable in the future it will cause a (minor) headache for someone in the future